cypress/e2e/application/index.cy.js

```diff
@@ -1,33 +1,37 @@
-import config from '../../support/config/services';
+import loadableConfig from '#app/lib/config/services/loadableConfig';
 import appConfig from '../../../src/server/utilities/serviceConfigs';
-import serviceHasPageType from '../../support/helpers/serviceHasPageType';
-import ampOnlyServices from '../../support/helpers/ampOnlyServices';
+
+const PAGES_WITHOUT_SERVICE_WORKER = [
```

Comment:
I think we could rename this: 
```suggestion
const PUBLIC_SERVICES = [
```